packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordModal.tsx

```diff
@@ -138,11 +134,12 @@ const DxtradePasswordModal = () => {
         isLoading,
         onSubmit,
         password,
+        show,
         dxtradePlatform,
         error?.error?.code,
     ]);
 
-    if (status === 'error' && error?.error?.code !== 'PasswordError') {
+    if (status === QueryStatus.ERROR && error?.error?.code !== 'PasswordError') {
```

Comment:
you can define `ErrorCodes` in the constants file so we can have all the codes in one place for better manupulation